CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Infrastructure
 * Replaced `pyflakes` with `ruff` for linting
+* Automatically set copyright date when generating the docs
```

Comment:
I think you accidentally started from https://github.com/reef-technologies/b2-sdk-python/pull/282 branch
btw, was that ever submitted upstream?

Reply:
yes, you are right. i did by mistake.  it is actually my branch and should be already merged, it was reviewed. I think we can merge this commit with this PR. ok?

Reply:
I don't think that we can - basically workflow is, you create PR in reef, get it reviewed, then recreate it for upstream (backblaze repo).
Also we don't want stuff to get stuck on some other changes.

I did that a lot recently and it was no fun;p

Reply:
ok, i can rebase (i guess this is the correct term) and resubmit my branch. that would be the reasonable approach?

Reply:
little to late for a rebase if it was already submitted for review (the golden rule of rebasing ;) )
thankfully it is not that big change that got bundled in here - so we can just ignore it I guess for now

whatever we do -  please create PR for date change to upstream